Fix sticky header widths regression in IndexTable (#9250)
Shopify/polaris: .changeset/heavy-pumpkins-rush.md

---
'@shopify/polaris': patch
---

> Fix regression where sticky header column widths are incorrect after bulk selection in IndexTable.

Author: ```suggestion
Fixed regression where sticky header column widths were incorrectly resized after bulk selection in the `IndexTable`.
```
